doc/notebooks/example_notebook.ipynb

```diff
@@ -1,28 +0,0 @@
-{
```

Comment:
You really need to do the `git checkout` command to get the original version of this file or copy-paste the content from the _File changes_ here and place it to the empty file. Committing an empty file says I want this file to be empty.

Reply:
I don't see anything expect the last character (last two actually), but there must be something different. Maybe the line endings. Try to get the original file from Git:

```
git checkout upstream/master -- .../the/file/path/file/name.py
```

Some other things than upstream/master may work too. Ideally, _without_ updating to latest master branch first.